nuopc.runconfig

```diff
@@ -278,19 +278,19 @@ MED_attributes::
 ::
 
 CLOCK_attributes::
-     atm_cpl_dt = 3600
+     atm_cpl_dt = 99999       #not used
```

Comment:
```suggestion
     atm_cpl_dt = 99999 #not used
```

Reply:
Resolved in this commit [5899c1c](https://github.com/COSIMA/MOM6-CICE6/pull/56/commits/5899c1c4b8c0c972a5fb526147031e1fea3f5fd5)

Reply:
Resolved in this commit [5899c1c](https://github.com/COSIMA/MOM6-CICE6/pull/56/commits/5899c1c4b8c0c972a5fb526147031e1fea3f5fd5)